tests/acceptance/28_inform_testing/.gitignore

```diff
@@ -0,0 +1 @@
+*.actual
```

Comment:
Shouldn't these files be removed automatically?

Reply:
They could be, yes. I can add that to the `init` bundle of `common.cf.sub`.

Reply:
But why not just ignore them as well? You mean remove them after each test is run and the diff is checked?

Reply:
It was very helpful while working on the tests to have the files there after a run, to make sure it looked "ok". So maybe I'm not understanding your comment. :)

Reply:
To me, seeing them in `git status` would be a nice indication of "something went wrong".

Reply:
sure, that would preserve my way of working as well, if they exist then there was a diff. right on.